README.md

```diff
@@ -168,14 +168,26 @@ Comming soon. We are contributing to support this now. See: https://github.com/f
 
 ### Run Fluter app
 
-Wayland compositor (weston) must be running before running the program when you use the Wayland backend.
+#### for Wayland backend
+
+Wayland compositor (weston) must be running before running the program.
 
 ```Shell
 $ ./flutter-client ./sample/build/linux/x64/release/bundle
 ```
 
-#### Note
-You need to run this program by a user who has the permission to access the input devices(/dev/input/xxx), if you use the DRM backend. Generally, it is a root user or a user who belongs to an input group.
+#### for DRM backend
+
+You should use an environment where x11 is not running. To turn off x11, switch to the CUI environment by pressing such as Shift+Alt+F3.
```

Comment:
You need to switch from GUI which is running X11 or Wayland to the Character User Interface (CUI). In addition, `FLUTTER_DRM_DEVICE` must be set properly. The default value is `/dev/dri/card0`.

```Shell
$ Shift + Alt + F3 # Switching to GUI
$ FLUTTER_DRM_DEVICE="/dev/dri/card1" ./flutter-drm-backend ./sample/build/linux/x64/release/bundle
```